src/rpcnet.cpp

```diff
@@ -443,3 +443,79 @@ Value getnetworkinfo(const Array& params, bool fHelp)
     obj.push_back(Pair("localaddresses", localAddresses));
     return obj;
 }
+
+Value setban(const Array& params, bool fHelp)
+{
+    string strCommand;
+    if (params.size() >= 2)
+        strCommand = params[1].get_str();
+    if (fHelp || params.size() < 2 ||
+        (strCommand != "add" && strCommand != "remove"))
+        throw runtime_error(
+                            "setban \"node\" \"add|remove\" (bantime)\n"
+                            "\nAttempts add or remove a IP from the banned list.\n"
+                            "\nArguments:\n"
+                            "1. \"ip\"       (string, required) The IP (see getpeerinfo for nodes ip)\n"
+                            "2. \"command\"  (string, required) 'add' to add a IP to the list, 'remove' to remove a IP from the list\n"
+                            "1. \"bantime\"  (numeric, optional) time in seconds how long the ip is banned\n"
```

Comment:
Right. This is a good point. It's actually the cmd arg -bantime. Will add this information.